config/routes.rb

```diff
@@ -1,10 +1,7 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Reveal health status on /up that returns 200 if the app boots with no exceptions, otherwise 500.
-  # Can be used by load balancers and uptime monitors to verify that the app is live.
+  root "users#index"
+  resources :users, only [:index, :show] do
```

Comment:
- [ ] You made a mistake here. There should be a ":" after "only".

  ```
  resources :users, only: [:index, :show] do
  ```

  ![image](https://github.com/Graciano1997/technology_blog_app/assets/81468675/305e7764-f2dd-429a-805b-81385529dffd)